core/db/entities.py

```diff
@@ -62,42 +51,21 @@ class Package(db.Model):
 
     __tablename__ = "package_dim"
 
-    id = sqla.Column(
-        GUID(),  # this should be UUIDType once using Postgres
-        default=uuid.uuid4,
-        primary_key=True,
-    )
+    id = sqla.Column(GUID(), default=uuid.uuid4, primary_key=True)  # this should be UUIDType once using Postgres
     package_name = sqla.Column(sqla.String, nullable=False, unique=True)
-    # TODO: should we limit string length here?
+
+    # TODO: should we limit string length here, for example?
     fund_type_id = sqla.Column(sqla.String, nullable=False)
+
     # TODO: Check that we need organisation directly referenced from Package SEPARATELY from the organisation
     #  lookups via each contact fk.
-    organisation_id = sqla.Column(
-        sqla.String,
-        sqla.ForeignKey("organisation_dim.id"),
-        nullable=False,
-    )
+    organisation_id = sqla.Column(sqla.String, sqla.ForeignKey("organisation_dim.id"), nullable=False)
+
     # TODO: Generic name definition in model, should we be more specific?
-    name_contact_id = sqla.Column(
-        sqla.String,
-        sqla.ForeignKey("contact_dim.id"),
-        nullable=False,
-    )
-    project_sro_contact_id = sqla.Column(
-        sqla.String,
-        sqla.ForeignKey("contact_dim.id"),
-        nullable=False,
-    )
-    cfo_contact_id = sqla.Column(
-        sqla.String,
-        sqla.ForeignKey("contact_dim.id"),
-        nullable=False,
-    )
-    m_and_e_contact_id = sqla.Column(
-        sqla.String,
-        sqla.ForeignKey("contact_dim.id"),
-        nullable=False,
-    )
+    name_contact_id = sqla.Column(sqla.String, sqla.ForeignKey("contact_dim.id"), nullable=False)
+    project_sro_contact_id = sqla.Column(sqla.String, sqla.ForeignKey("contact_dim.id"), nullable=False)
+    cfo_contact_id = sqla.Column(sqla.String, sqla.ForeignKey("contact_dim.id"), nullable=False)
+    m_and_e_contact_id = sqla.Column(sqla.String, sqla.ForeignKey("contact_dim.id"), nullable=False)
```

Comment:
could also do this tidying up for the ProjectProgress class for consistency? I just copied and pasted but I think it looks better when it's all on one line like this now that we have the longer line length

Reply:
yeah, I can do that in this PR